.github/workflows/build-binaries.yaml

```diff
@@ -15,14 +15,16 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        os: [ubuntu-latest, windows-latest, macos-latest, macos-14]
```

Comment:
Is there a more explicit way of requesting apple silicon? This isn't obvious :P If you can't we should add a comment to explain...

Reply:
Good idea, I'll update it.